packages/wallets/src/features/cfd/screens/Jurisdiction/JurisdictionScreen.scss

```diff
@@ -6,12 +6,18 @@
     justify-content: center;
     margin: auto 3rem;
     height: 75vh;
-    width: 85vw;
```

Comment:
If its for the dynamic leverage screen, you can style it inside `DynamicLeverageScreen.scss`, only the screen should control the width/height of the modal imo

Reply:
Updated.